servicecontrol/plugins/index.md

```diff
@@ -13,15 +13,32 @@ NOTE: When ServiceControl is introduced into an existing environment the standar
 
 Plugins collect information from NServiceBus and are deployed together with each NServiceBus endpoint. These plugins are optional from the perspective of the NServiceBus framework itself (they are not required by the endpoint), but they are required in order to collect the information that enables ServiceControl (and its clients) to provide the relevant functionality for each plugin.
 
+## Functionalities
 
-## Installing and Deploying
+ServiceControl provide monitoring by analyzing configured audit and fault queues. Based on audit and fault messages it can extract information like: endpoint names, queue names, exception stack traces etc. Based on that information ServiceControl stores them in built-in database and exposes them to 3rd party tools.
+
+Plugins allow to extend given functionalities by providing additional data such as:
+ - [Heartbeat Plugin](heartbeat.md) - report if endpoint has connectivity with ServiceControl by sending heartbeat messages.
+ - [Saga Plugin](saga-audit.md) - report saga related data to allow for better analysis.
+ - [Custom Check Plugin](custom-checks.md) - allow for writing custom checks and sending information about its failure or success.
+
+## Configuring endpoint to be monitored by ServiceControl
+
+To allow for ServiceControl to monitor endpoints messages the following steps should be followed:
+
+1. ServiceControl should be [installed](/servicecontrol/installation.md) and at least one instance should be configured using the same Transport as is used by endpoints
+2. For every endpoint that messages should be monitored by ServiceControl [configure audits](/nservicebus/operations/auditing.md#configuring-auditing). The audit queue should match audit queue configured for ServiceControl.
+3. For every endpoint that failed messages should be monitored by ServiceControl [configure error messages](/nservicebus/errors/#configure-the-error-queue). The erorr queue should match error queue configured for ServiceControl.
+
+
+## Installing and Deploying Plugins
 
 The ServiceControl plugins are deployed with the endpoints they are monitoring. It is possible add a plugin to an endpoint during development, testing, or production:
 
  * During development, add the relevant plugin NuGet package to the endpoint's project in Visual Studio using the NuGet.
  * When in production, add the plugin DLLs to the BIN directory of the endpoint, and restart the endpoint process for the changes to take effect and the plugin to be loaded.
 
-NOTE: For NServiceBus version-dependent requirements for each plugin, review the "Dependencies" section in the NuGet Gallery page for the specific plugin. 
+NOTE: For NServiceBus version-dependent requirements for each plugin, review the "Dependencies" section in the NuGet Gallery page for the specific plugin.
```

Comment:
Is this actually necessary? Each plugin is named after the core version that it targets.